packages/app/components/token-picker/TokenPicker.tsx

```diff
@@ -118,32 +130,31 @@ const SearchBar = forwardRef<HTMLInputElement, SearchBarProps>(
 );
 SearchBar.displayName = "Token Picker Search";
 
-const CommonTokens = () => (
+const CommonTokens = ({ onTokenSelect }: CommonTokensProps) => (
   <div className="mt-5">
     <p className="text-xs font-semibold text-em-low">Common tokens</p>
     <div className="flex flex-wrap mt-2 gap-2">
-      {TOKEN_PICKER_COMMON_TOKENS.map((token: Token) => (
-        <RadioButton
-          checked={false}
+      {TOKEN_PICKER_COMMON_TOKENS.map((token: TokenFromTokenlist) => (
+        <ChipButton
           id={token.address}
           key={token.address}
           name={token.name}
-          onChange={() => {}}
-          value={token.name}
+          onClick={() => onTokenSelect(token)}
+          size="sm"
         >
           <TokenIcon token={token} />
           <p className="font-semibold ml-1.5">{token.symbol}</p>
```

Comment:
Same here let's use `BodyText` component.

```suggestion
<BodyText className="ml-1.5">{token.symbol}</BodyText>
```

Reply:
Isn't this the same component as above?

Maybe we should create `ChipTokenButton` and have only one of this. Only one place to change.

Reply:
It's not exactly the same, I'll end up deleting the other ChipTokenButton as we're not needing it and we can solve the approach with either `ChipButton` or plain `Button` :D